.github/workflows/integration-tests.yml

```diff
@@ -811,6 +1160,30 @@ jobs:
           matrix-aggregator-status: ${{ needs.eth-smoke-tests-matrix.result }}
         continue-on-error: true
 
+  eth-smoke-tests-ccip:
+    if: always()
+    runs-on: ubuntu-latest
+    name: ETH Smoke Tests CCIP
+    needs: eth-smoke-tests-matrix-ccip
+    steps:
+      - name: Check smoke test matrix status
+        if: needs.eth-smoke-tests-matrix-ccip.result != 'success'
+        run: |
+          echo "ETH Smoke Tests CCIP: ${{ needs.eth-smoke-tests-matrix-ccip.result }}"
+          exit 1
+      - name: Collect Metrics
+        if: always()
+        id: collect-gha-metrics
+        uses: smartcontractkit/push-gha-metrics-action@d9da21a2747016b3e13de58c7d4115a3d5c97935 # v3.0.1
+        with:
+          id: ${{ env.COLLECTION_ID }}-matrix-results-ccip
+          org-id: ${{ secrets.GRAFANA_INTERNAL_TENANT_ID }}
+          basic-auth: ${{ secrets.GRAFANA_INTERNAL_BASIC_AUTH }}
+          hostname: ${{ secrets.GRAFANA_INTERNAL_HOST }}
+          this-job-name: ETH Smoke Tests CCIP
+          matrix-aggregator-status: ${{ needs.eth-smoke-tests-matrix-ccip.result }}
+        continue-on-error: true
```

Comment:
is there a reason for not making it part of `eth-smoke-tests` job?

Reply:
CCIP tests shouldn't be run on every single PR. We can gate them to run only on changes in the `ccip` directories. For now I'm leaving them running on each PR to stabilize this transition, but will separate them more clearly soon.